<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMaterialAsset</class>
 <widget class="QWidget" name="PMaterialAsset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Material properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Unit 0:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Unit0ComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Unit 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Unit1ComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Unit 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Unit2ComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Unit 3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Unit3ComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
